client/src/app/admin/components/sites/Sites.tsx

```
@@ -49,8 +41,7 @@ export function Sites() {
       const lowerSearchQuery = searchQuery.toLowerCase();
       return (
         site.domain.toLowerCase().includes(lowerSearchQuery) ||
-        (site.organizationOwnerEmail &&
-          site.organizationOwnerEmail.toLowerCase().includes(lowerSearchQuery))
+        (site.organizationOwnerEmail && site.organizationOwnerEmail.toLowerCase().includes(lowerSearchQuery))
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)